doc/user-guide/oauthn_token_exchange.md

```diff
@@ -0,0 +1,124 @@
+## OAuthN Token exchange support
+
+The current release of Indigo IAM implements an essential set of feature of the
+[Token Exchange OAuth specification](https://tools.ietf.org/html/draft-ietf-oauth-token-exchange-05).
+Indigo IAM supports only impersonation semantic: delegation is not yet supported.
+
+### Configuration
+Clients that request Token Exchange must be configured with
+the `urn:ietf:params:oauth:grant-type:token-exchange` grant type enabled.
+
+### Usage
+A client who wants to exchange an access token with a new one, must send a request to the `/token` endpoint,
```

Comment:
with a new one (or a couple of new tokens, in case a refresh token is requested)